external/esp_idf_port/spi_flash/include/esp_partition.h

```diff
@@ -36,7 +36,7 @@
 #include <stdint.h>
 #include <stdbool.h>
 #include <stddef.h>
-//#include "esp_err.h"
+#include <arch/chip/esp_err.h>
```

Comment:
why include esp_err.h here, while it was commented in the old file?

Reply:
Removed